Source/DafnyPipeline.Test/Trivia.cs

```diff
@@ -147,7 +147,6 @@ ensures true
       }
 
       AreAllTokensOwned(program);
-      Assert.Equal(9, count); // Sanity check
```

Comment:
Why does this need to be removed?

Reply:
Because the DefaultModule now has a filename in its StartToken, the traversal doesn't go deeper than that Node and count only gets up to 1. This is fine though. I'm not sure why `Assert.Equal(9, count)` was in there because it wasn't asserting any required behavior.